Interactive doesn't work after adding nodes.json

```
Traceback (most recent call last):
  File "/Users/yzguy/Projects/routedbits/dn42-peers/interactive.py", line 268, in <module>
    main(args)
  File "/Users/yzguy/Projects/routedbits/dn42-peers/interactive.py", line 244, in main
    peer_errors = list(validate(peer))
                       ^^^^^^^^^^^^^^
TypeError: validate() missing 1 required positional argument: 'peer'
make: *** [peering] Error 1
```

--- a/interactive.py
+++ b/interactive.py
@@ -99,6 +99,8 @@ def main(args):
         except (IndexError, TypeError, ValueError):
             output.fail('ERROR: Not a valid selection, try again')
 
+    node_type = next(node['type'] for node in nodes if node['hostname'] == router)
+
     # ASN
     while True:
         try:
@@ -231,7 +233,7 @@ def main(args):
         if public_key:
             wireguard['public_key'] = public_key
 
-        if not (errors := validations.validate_wireguard(wireguard)):
+        if not (errors := validations.validate_wireguard(wireguard, require_ipv4=(node_type=='ipv4'))):
             break
 
         for error in errors:
@@ -241,7 +243,7 @@ def main(args):
     print()
 
     # Final validation as a whole
-    peer_errors = list(validate(peer))
+    peer_errors = list(validate(node_type, peer))
     for peer_error in peer_errors:
         output.fail(peer_error)
 

--- a/routers/router.mum1.yml
+++ b/routers/router.mum1.yml
@@ -1,1 +1,12 @@
 ---
+- name: CDUBS-IN
+  asn: 4242420566
+  ipv6: fe80::566:20
+  multiprotocol: true
+  extended_nexthop: true
+  sessions:
+    - ipv6
+  wireguard:
+    remote_address: dn20.in.surgebytes.com
+    remote_port: 30207
+    public_key: tDYc6/7gZCW9gFOVI64dBC5g72qd6fhocNYKe1MbiiI=

